<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KullaniciDialogForm</class>
 <widget class="QDialog" name="KullaniciDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>470</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bitstream Vera Sans Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/check.png</normaloff>:/images/ui/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ui/remove.png</normaloff>:/images/ui/remove.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Yetkilendirme</string>
   </property>
   <widget class="QCheckBox" name="StokcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stok işlemleri yapabilir</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="KasacheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kasa giriş/çıkış yapabilir</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="IadecheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>İade alabilir</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="AyarcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ayarları değiştirebilir</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CaricheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cari işlemleri yapabilir</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>8</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kullanıcı adı</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="UserNamelineEdit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Şifre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PasswordlineEdit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Şifre tekrar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="PasswordlineEdit_2">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Adı</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="AdlineEdit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Soyadı</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="SoyadlineEdit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cep No</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="CepNolineEdit">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UserNamelineEdit</tabstop>
  <tabstop>PasswordlineEdit</tabstop>
  <tabstop>PasswordlineEdit_2</tabstop>
  <tabstop>AdlineEdit</tabstop>
  <tabstop>SoyadlineEdit</tabstop>
  <tabstop>CepNolineEdit</tabstop>
  <tabstop>KasacheckBox</tabstop>
  <tabstop>IadecheckBox</tabstop>
  <tabstop>StokcheckBox</tabstop>
  <tabstop>CaricheckBox</tabstop>
  <tabstop>AyarcheckBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
